### DISCUSSION TITLE
Welcome to congacomic.github.io Discussions!

### DISCUSSION THREAD

**Discussion by mbwhite** [congacomic/congacomic.github.io#71] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by EdMoffatt** [comment]
HELLO CONGAMMUNITY!

I'm Ed, and I draw cubes with faces on, as directed by comic ideas from this community (mostly Andy Hurt if we can manage to sign his talents before the transfer deadline)

**Comment by jt-nti** [comment]
Hello @EdMoffatt! Not sure how best to use this spangly new discussion thing so I'm just going to throw in a poorly thought through idea while we wait for Andy...

Has there been an NFT comic yet?!
